packages/react-native-renderer/src/ReactFabricHostConfig.js

```diff
@@ -390,7 +398,16 @@ export function cloneUnhiddenInstance(
   props: Props,
   internalInstanceHandle: Object,
 ): Instance {
-  throw new Error('Not yet implemented.');
+  const node = instance.node;
+  const updatePayload = ReactNativeAttributePayload.diff(
+    {...props, style: [props.style, {display: 'none'}]},
```

Comment:
What does spread compile to?

Reply:
```
Object.assign({}, props, { style: [props.style, { display: "none" }] }),
```

which is polyfilled https://github.com/facebook/react-native/blob/cc90c20bf60c72fc1b42f3306636847ce7f478e5/Libraries/polyfills/Object.es6.js#L15